packages/wonder-blocks-modal/components/modal-dialog.js

```diff
@@ -77,6 +77,7 @@ export default class ModalDialog extends React.Component<Props> {
                             {below && <View style={styles.below}>{below}</View>}
                             <View
                                 role="dialog"
+                                aria-modal={true}
```

Comment:
I think this should be `aria-model="true"` to match other boolean aria props as defined in https://github.com/Khan/wonder-blocks/blob/master/packages/wonder-blocks-core/util/types.js#L95-L142.  Also, please add an `aria-modal` entry to `AriaProps` in that file.